app/views/configuration/_ui_1.html.haml

```diff
@@ -77,6 +77,18 @@
               miqSelectPickerEvent("#{item_per_page[1]}", "#{url}")
         %fieldset
           %h3
+            = _('Topology View Settings')
+          .form-group
+            %label.col-md-3.control-label
+              = _("Default Maximum Items")
+            .col-md-8
+              = select_tag(_("topology_max_objects"),
+                           options_for_select([[N_("Unlimited"), 0]].concat(PPCHOICES), @edit[:new][:topology][:max_objects]),
```

Comment:
@zeari it seems you could create a `PPCHOICES_UNLIMITED` (or `PPCHOICES3`?) [here](https://github.com/ManageIQ/manageiq-ui-classic/blob/3679925/app/helpers/ui_constants.rb#L58). Otherwise other engineers will hardly discover this additional value we're introducing. Also I think over 200 items is already "unlimited" for the topology.